charts/seed-bootstrap/charts/monitoring/templates/config.yaml

```diff
@@ -32,6 +32,9 @@ data:
         regex: __meta_kubernetes_pod_label_(.+)
 
     - job_name: garden
+      scheme: https
```

Comment:
Can we be sure, that all scraped components support https?
Should we maybe have one rule for https (default) and one for http (pods labelled with `prometheus.io/scheme=http`) or something similar?

Reply:
We thought about this, but wanted to keep the config as is until we need to support this (which will probably happen very soon). If you prefer I can add this config now.

Reply:
Fair enough, let's add it later once we need it 👍